### DISCUSSION TITLE
Template for showcasing components

### DISCUSSION THREAD

**Discussion by JaeAeich** [elixir-cloud-aai/cloud-components#92] [Ideas]
### Description

Most of the packages as of now use simply use `HTML` to showcase their Web Component, if so wouldn't it be better to have a standardized way to display them? Maybe creating a package and styles which injects `js` and `CSS` into this ' index.html` and to give a standard overview of the component.

`Storybook` is the obvious solution to this but the problem with this is that it's too exhaustive for normal users/code reviewers to judge the package at a glance. 

## Pros
- A standard way to showcase the components
- A clean and consistent workflow or at least UI preview

## Cons
- Might be overkill
- Can be taxing as it is a subset of what (if implemented) the `storybook` will do.

**Comment by uniqueg** [comment]
Just seeing this @JaeAeich - what did you guys eventually agree on here?

**Reply by JaeAeich** [reply]
We decided againsts it, storybook is basically a way to make interactively documentation for component and but our approach of design tokens and CSS variables already produced a similar result, ie if you look at the [documentation](https://cloud-components.2.rahtiapp.fi/) you can play around with it. Yes its not as interactive as storybook but its a sweet middle ground and way less overhead.
